test/Negative.test.js

```diff
@@ -1,3 +1,4 @@
+'use strict';
```

Comment:
this shouldn't be here, be careful that only the changes you made are in this diff, otherwise you're going to revert changes from other diffs!